linkerd/tls/src/main/scala/io/l5d/clientTls/boundPath.scala

```diff
@@ -32,6 +32,7 @@ case class boundPath(caCertPath: Option[String], names: Seq[NameMatcherConfig],
       } match {
         case None if (strict.getOrElse(true)) =>
           throw new MatcherError(s"Unable to match ${path.show} with available names: ${(names.map(_.prefix)).mkString(",")}")
+        case default => default
```

Comment:
@adleong we'll never get to 52/53 if this returns None...  We may want to figure out a better way to do this moving forward (so that we're not just throwing an exception during stack construction) but for now I'm happy just to fix this as-is.